smoketest/cmd/scans/main.go

```diff
@@ -22,7 +22,7 @@ const protocoll = "unix"
 func init() {
 	address = os.Getenv("OSPD_SOCKET")
 	if address == "" {
-		address = "/var/run/ospd/ospd.sock"
+		address = "/var/run/ospd/ospd-openvas.sock"
```

Comment:
```suggestion
		address = "/run/ospd/ospd-openvas.sock"
```

Reply:
I used a sed command to change every case instead of using suggestion; I hope you don't mind.